Content.Server/Strip/StrippableSystem.cs

```diff
@@ -76,6 +82,27 @@ private void OnStripHandcuffMessage(EntityUid uid, StrippableComponent component
             }
         }
 
+        private void OnStripEnsnareMessage(EntityUid uid, StrippableComponent component, StrippingEnsnareButtonPressed args)
+        {
+            if (args.Session.AttachedEntity is not {Valid: true} user)
+                return;
+
+            if (TryComp<EnsnareableComponent>(component.Owner, out var ensnared))
+            {
+                foreach (var entity in ensnared.Container.ContainedEntities)
+                {
+                    if (!TryComp<EnsnaringComponent>(entity, out var ensnaring))
+                        return;
```

Comment:
```suggestion
                        continue;
```

Probably use an GetEntityQuery<T> here too.

Reply:
Added the continue, not sure how to utilize the GetEntityQuery here, it either makes a double foreach loop or it won't let me put in what I want.